foosball/static_src/js/foosball.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready(function() {
```

Comment:
Should we protect the dollar like [this](https://learn.jquery.com/plugins/basic-plugin-creation/#protecting-the-alias-and-adding-scope)? Also looking at the content, this file could have a name `foosball_stats.hs`?